docs/CONFIGURATION.md

```diff
@@ -114,7 +116,7 @@ For more information see the [Go Cloud Development Kit docs for gcsblob](https:/
 
 ##### Local filesystem Storage
 
-For more information see the [Go Cloud Development Kit docs for fileblob](https://godoc.org/gocloud.dev/blob/fileblob). This is the default if no other provider is specified. Use `CLOUD_PROVIDER=file` to read the following environmental variables:
+For more information see the [Go Cloud Development Kit docs for fileblob](https://godoc.org/gocloud.dev/blob/fileblob). This is the default if no other provider is specified. Use `CLOUD_PROVIDER=local` to read the following environmental variables:
```

Comment:
Same comment here about `environmental`. I recommend doing a replace all in the file.

Reply:
Can do. Should have looked more closely when I copied/pasted it.